markets/perps-market/contracts/modules/AsyncOrderModule.sol

```diff
@@ -191,9 +189,25 @@ contract AsyncOrderModule is IAsyncOrderModule {
         PerpsMarket.Data storage perpsMarket = PerpsMarket.loadValid(asyncOrder.marketId);
         perpsMarket.updatePositionData(newPosition);
 
+        // taken from asyncOrder to use it in event (after order is reset)
+        bytes32 trackingCode = asyncOrder.trackingCode;
+        uint128 accountId = asyncOrder.accountId;
```

Comment:
maybe move this to above line 167 so you don't have to storage read multiple times